src/DssSpell.sol

```diff
@@ -35,14 +27,9 @@ interface ProxyLike {
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/ab5af3e3179d17d7a91a97ac0fec234bec0e26bb/governance/votes/Executive%20vote%20-%20February%2022%2C%202024.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/94c78956105c9ff0cb5aa3662e371f75bfb4fa5f/governance/votes/Executive%20Vote%20-%20March%206%2C%202024.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2024-02-22 MakerDAO Executive Spell | Hash: 0xfbaf5ce577a6f054a014681eacff991c409f625b8146e8f8bdcd4f8f58ab386d";
-
-    // Set office hours according to the summary
-    function officeHours() public pure override returns (bool) {
-        return false;
-    }
```

Comment:
I know that office hours are [on by default](https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssAction.sol#L42-L44), but let's make it _explicit_ for readability and comfort of external reviewers. 

In the past, we removed `officeHours` override only if a spell did not fit into the contract size. Which I don't think is the case this time.